### PR TITLE
Cij dates

### PR DESCRIPTION
**Pull request by Jennit07**
Derived cij dates removed, cij dates added into acute, maternity and mental health for all years. Tested 1617 and 1819 syntax, all seems to work fine.

### CODE REVIEW

File: 2015-16/C01 Make Episode File.sps [hunk 1]
```diff
@@ -369,7 +362,38 @@ aggregate
     /cij_ppa = Max(PPA).
 
 sort cases by chi keydate1_dateformat.
-
+<<<<<<< HEAD
```
Comment (Moohan) [inline 2015-16/C01 Make Episode File.sps:365]: This shouldn't be here. Was there a merge conflict?

File: 2015-16/C01 Make Episode File.sps [hunk 2]
```diff
@@ -373,7 +366,7 @@ sort cases by chi keydate1_dateformat.
 
 *  We are not currently including Social Care data for some years but we still want the variables for consistency.
 * Social Care.
- * String
+String
```
Comment (Moohan) [inline 2015-16/C01 Make Episode File.sps:369]: This and the CH variables below should not need to be added in the 1516 file

File: 2016-17/C01 Make Episode File.sps [hunk 3]
```diff
@@ -354,11 +364,11 @@ aggregate
     /Break chi cij_marker
     /cij_ppa = Max(PPA).
 
-sort cases by chi CIJ_start_date.
+sort cases by chi keydate1_dateformat.
 
 *  We are not currently including Social Care data for some years but we still want the variables for consistency.
 * Social Care.
- * String
+ String
```
Comment (Moohan) [inline 2016-17/C01 Make Episode File.sps:371]: Same as 1516 the CH and sc_send_lca should already exists

File: 2015-16/C01 Make Episode File.sps (hunk 1 above)
Comment (Jennit07) [reply 2015-16/C01 Make Episode File.sps]: Line removed

File: 2015-16/C01 Make Episode File.sps (hunk 2 above)
Comment (Jennit07) [reply 2015-16/C01 Make Episode File.sps]: CH variables added back in 1516 file, not too sure about numeric variables. Can you please double check what i have added back in/deleted is correct

File: 2016-17/C01 Make Episode File.sps (hunk 3 above)
Comment (Jennit07) [reply 2016-17/C01 Make Episode File.sps]: CH variables added back in 1617 file, not too sure about numeric variables. Can you please double check what i have added back in/deleted is correct

**Pull request merged**